### DISCUSSION TITLE
Mint MoodNFT a second time

### DISCUSSION THREAD

**Discussion by Totiruzi** [Cyfrin/foundry-full-course-cu#2699] [General]
Trying to mint the MoodNFT a second time. 
I actually minted the mood NFT's imported the NFT and Flipped the Mood too. 
But for some reasons. I restarted Anvil on my local host, deployed and minted. But this time around I could not import to metalmask with an error

![image](https://github.com/user-attachments/assets/29fb5e0e-b820-43fa-857a-653c754ef719)

I have tried editing the network by renaming, clearing the catch but still have the same error ownership details do not match
Please any idea, what to do?

Thanks )

**Comment by EngrPips** [comment]
Hello @Totiruzi, So I did the process and repeated it, and everything worked fine for me. So follow the steps below, and let's see if it works for you.

=> Restart the `Anvil` blockchain in your terminal 
=> Go to the `advance setting` in your metamask and click on `clear activity tab data` to clear any data from the previous `Anvil blockchain' that metamask has stored. The image below should help you understand what I mean

<img width="1440" alt="Screenshot 2024-10-17 at 11 37 03" src="https://github.com/user-attachments/assets/af382e78-a727-4b9e-add9-6816c30a2667">

=> Deploy a new NFT contract 
=> Mint on the new NFT contract
=> Go and add your minted NFT to metamask. 

Following the steps above, you should be able to add your NFT to metamask when you start a new `Anvil` chain.

**Reply by Totiruzi** [reply]
@EngrPips 
Wala!!! 
I was almost giving up, I tried the steps you specified before a numbers of times could not still import.
**Magic**
I decided to restart my computer and follow the steps once again and just like WALA I imported the mood to my metemask )

Thanks a whole bunch )
I really appreciate      )

**Reply by EngrPips** [reply]
You are welcome, Friend. Keep crushing it.

**Comment by Totiruzi** [comment]
How do you mark an answer correct @EngrPips )

**Reply by EngrPips** [reply]
You can't for this discussion because it is not `question and answer` but it is just a mere `discussion`

**Reply by Totiruzi** [reply]
Ah Ok!!!
Thanks buddy )

**Reply by EngrPips** [reply]
Anytime, Friend.
